### PR TITLE
Proof of concept config for payments-service

### PR DESCRIPTION
**Pull request by kumar303**
https://github.com/mozilla/payments-service/

This still relies on symlinks (for now).

### CODE REVIEW

**Comment by kumar303** [comment]
@andymckay r?

This requires https://github.com/mozilla/payments-service/pull/1

File: docker-compose.yml [hunk 1]
```diff
@@ -1,3 +1,17 @@
+paymentsservice:
```
Comment (andymckay) [inline docker-compose.yml]: we could just call it service or something

**Comment by andymckay** [comment]
r+ yay docker

File: docker-compose.yml (hunk 1 above)
Comment (kumar303) [inline docker-compose.yml]: I like that

**Pull request merged**